test/src/test_trtri.cpp

```diff
@@ -0,0 +1,90 @@
+/// @file test_trtri.cpp
+/// @brief Test TRTRI
+//
+// Copyright (c) 2022, University of Colorado Denver. All rights reserved.
+//
+// This file is part of <T>LAPACK.
+// <T>LAPACK is free software: you can redistribute it and/or modify it under
+// the terms of the BSD 3-Clause license. See the accompanying LICENSE file.
+
+#include <catch2/catch.hpp>
+#include <plugins/tlapack_stdvector.hpp>
+#include <tlapack.hpp>
+#include <testutils.hpp>
+#include <testdefinitions.hpp>
+
+using namespace tlapack;
+
+TEMPLATE_LIST_TEST_CASE("TRTRI is stable", "[trtri]", types_to_test)
+{
+    srand(1);
+
+    using matrix_t = TestType;
+    using T = type_t<matrix_t>;
+    using idx_t = size_type<matrix_t>;
+    typedef real_type<T> real_t;
+
+    Uplo uplo = GENERATE(Uplo::Lower, Uplo::Upper);
+    idx_t n = GENERATE(1, 2, 6, 9);
+
+    const real_t eps = uroundoff<real_t>();
+    const real_t tol = 1.0e2 * n * eps;
+
+    std::unique_ptr<T[]> A_(new T[n * n]);
+    std::unique_ptr<T[]> C_(new T[n * n]);
+
+    auto A = legacyMatrix<T, layout<matrix_t>>(n, n, &A_[0], n);
+    auto C = legacyMatrix<T, layout<matrix_t>>(n, n, &C_[0], n);
+
+    // Generate random matrix in Schur form
+    for (idx_t j = 0; j < n; ++j)
+    {
+        for (idx_t i = 0; i < n; ++i)
+            A(i, j) = rand_helper<T>();
+        A(j, j) += tlapack::make_scalar<T>(n, 0);
+    }
+
+    lacpy(uplo, A, C);
+
+    DYNAMIC_SECTION("n = " << n)
```

Comment:
@thijssteel and @weslleyspereira: How do we print Upper or Lower? @meieraheidi prints n. But I am not sure how to print uplo. Can you help?

Reply:
#101 will allow us to do:

```c++
DYNAMIC_SECTION("n = " << n << ", " << uplo )
```

for an output, e.g., `n = 9, Upper`. I will finish that PR.